src/client/public/openfin/platform.json

```diff
@@ -0,0 +1,183 @@
+{
+    "devtools_port": 9090,
+    "splashScreenImage": "{*host_url*}/static/media/splash-screen.jpg",
+    "runtime": {
+      "arguments": "--v=1 --inspect",
+      "version": "14.78.48.3"
```

Comment:
should we bump this? It's a couple of versions behind the other (old API) manifests

Reply:
Definitely. Latest.